app/controllers/api/teams.php

```diff
@@ -1102,14 +1113,14 @@
         $dbForProject->purgeCachedDocument('users', $user->getId());
 
         if ($membership->getAttribute('confirm')) { // Count only confirmed members
-            Authorization::skip(fn () => $dbForProject->decreaseDocumentAttribute('teams', $team->getId(), 'total', 1, 0));
```

Comment:
Why can't we use the `decreaseDocumentAttribute()` method?

Reply:
We totally can, I think I defaulted to habit haha

Reply:
This one hasn't been updated.